sys/arduino/include/arduino.hpp

```diff
@@ -34,7 +34,7 @@ extern "C" {
 enum {
     INPUT,              /**< configure pin as input */
     OUTPUT,             /**< configure pin as output */
-    INPUT_PULLUP,       /**< configure pin as input with pull-up resistor */
+    INPUT_PULLUP        /**< configure pin as input with pull-up resistor */
```

Comment:
It's not a matter of taste:
`error: comma at end of enumerator list [-Wpedantic]`
and http://david.tribble.com/text/cdiffs.htm#C99-enum-decl